web/templates/includes/end_js.html

```diff
@@ -1,3 +1,4 @@
+<script src="/js/chart.min.js?<?=JS_LATEST_UPDATE?>"></script>
```

Comment:
Maybe we should have `/js/vendor` folder for both jQuery and Chart.js

Then we can exclude this folder in `.eslintignore`.

Reply:
Agree with it. 

I would prefer to use npm for version control in the future...